docs/openapi3.yaml

```diff
@@ -2956,3 +3138,70 @@ components:
         url:
           type: string
           description: Webhook url to post events
+    chaosExperiment:
+      required:
+        - name
+        - kind
+        - namespace
+      properties:
+        id:
+          description: Unique experiment identifier
+          type: string
+          format: uuid
+          readOnly: true
+        name:
+          type: string
+          minLength: 1
+          description: The name of the experiment.
+          example: Custom javascript for logging
+        kubeObject:
+          $ref: '#/components/schemas/kubeObject'
+        updated_at:
+          type: string
+          format: date-time
+          description: The date and time that the procesor file was updated.
```

Comment:
"procesor file" not relevant in this case

Reply:
done